sunpy/net/vso/vso.py

```diff
@@ -742,29 +742,13 @@ def create_getdatarequest(self, maps, methods, info=None):
         if 'email' not in info:
             info['email'] = 'sunpy'
 
-        # For the JSOC provider we need to make a DataRequestItem for each
-        # series, not just one for the whole provider.
-
-        # Remove JSOC provider items from the map
-        jsoc = maps.pop('JSOC', [])
-        # Make DRIs for everything that's not JSOC one per provider
-        dris = [self.make('DataRequestItem', provider=k, fileiditem={'fileid': v})
-                for k, v in maps.items()]
-
-        def series_func(x):
-            """ Extract the series from the fileid. """
-            return x.split(':')[0]
-
-        # Sort the JSOC fileids by series
-        # This is a precursor to groupby as recommended by the groupby docs
-        series_sorted = sorted(jsoc, key=series_func)
-        # Iterate over the series and make a DRI for each.
-        # groupby creates an iterator based on a key function, in this case
-        # based on the series (the part before the first ':')
-        for series, fileids in itertools.groupby(series_sorted, key=series_func):
-            dris.append(self.make('DataRequestItem',
-                                  provider='JSOC',
-                                  fileiditem={'fileid': list(fileids)}))
+        reqitems = []
+        for k in maps.keys():
```

Comment:
Might be a good opportunity to use some more descriptive names here.
```suggestion
        for provider in maps.keys():
```